extensions/wikia/Chat2/templates/ChatBanList.tmpl.php

```diff
@@ -0,0 +1,172 @@
+<!-- s:<?= __FILE__ ?> -->
+<!-- DISTRIBUTION TABLE -->
+<script type="text/javascript" charset="utf-8">
+$( function () {
```

Comment:
I don't think we should use this style in new code - creating inline javascript means this asset cannot be cached, and it also prevents Wikia from loading assets at the bottom of the `<body>` tag. This should probably live in its own file and use ResourceLoader to load the messages client side.